salt/__init__.py

```diff
@@ -141,3 +140,11 @@ def __define_global_system_encoding_variable__():
 import salt._logging  # isort:skip
 
 # pylint: enable=unused-import
+
+
+# When we are running in a 'onedir' environment, setup the path for user
+# installed packages.
+if hasattr(sys, "RELENV"):
+    sys.path = [
+        str(sys.RELENV / "extras-{}.{}".format(*sys.version_info[:2]))
+    ] + sys.path
```

Comment:
```suggestion
    sys.path.insert(0, str(sys.RELENV / "extras-{}.{}".format(*sys.version_info[:2])))
```
No extra lists created.